app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentUpdate.kt

```diff
@@ -36,6 +36,14 @@ class ScheduleAppointmentUpdate(
     }
   }
 
+  private fun appointmentFacilityLoaded(
+      model: ScheduleAppointmentModel,
+      event: AppointmentFacilitiesLoaded
+  ): Next<ScheduleAppointmentModel, ScheduleAppointmentEffect> {
+    val appointmentFacility = event.assignedFacility ?: event.currentFacility
```

Comment:
I am merging this for now, but can we add additional tests in the Mobius style to verify this behaviour? We should add similar tests to the effect handler as well.

Reply:
Will do